app/models/user.rb

```diff
@@ -1,4 +1,4 @@
-class Volunteer < ApplicationRecord
+class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
```

Comment:
can you add something like `enum role: [:Volunteer, :Leader]` as described in this link: https://stackoverflow.com/questions/37849085/rails-permissions-cancancan-abilities-show-only-what-user-creates
and also do the corresponding migration. It'll help for when we want to differentiate permissions

Reply:
done